rails/app/models/kaiju/component.rb

```diff
@@ -89,8 +89,8 @@ def self.generate_props_for_properties(properties)
       end
     end
 
-    def ast # rubocop:disable Metrics/MethodLength
-      info = ComponentInformation.info(type)
+    def ast # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
At some point it might be beneficial to adjust some of these defaults in the rubocop.yml we're hitting a hard ceiling on a lot of these.